Makefile

```diff
@@ -1,22 +1,22 @@
 install:
 	@printf "\e[0;32mGenerating RSA key to encrypt webtokens..\e[0m\n"
-	@mkdir -p var/jwt
+	@[ -d var/jwt ] || @mkdir -p var/jwt
```

Comment:
What does `[ -d var/jwt ]` do? It checks if the directory exists?

Reply:
Yes this is what IT does.

Reply:
Is it possible that the second "@" is wrong? I'm getting a command not found for "@mkdir". It works fine if I remove the at sign:

`@[ -d var/jwt ] || mkdir -p var/jwt`

Reply:
I think `@` is to make the command "silent", no?

Reply:
Yes, but I believe it applies to the entire line. The OR is part of the command, so you don't need to silence the mkdir again. Checkout the similar command in the osrm target on line 19:
```
@[ -d var/osrm ] || mkdir -p var/osrm
```